runtime/rasdump/javadump.cpp

```diff
@@ -2524,7 +2543,7 @@ void
 JavaCoreDumpWriter::writeHookSection(void)
 {
 	/* Write the section header */
-	_OutputStream.writeCharacters("0SECTION       HOOK subcomponent dump routine\n");
+	_OutputStream.writeCharacters("0SECTION       HOOK subcomponent dump routine NOTE: This data only represents the period since the previous javacore\n");
```

Comment:
I think this note should appear on a separate line from the `0SECTION`; javacore files are purposely structured to make parsing easier - this might negate some of that.

Reply:
Would adding a line right below the "0SECTION       HOOK subcomponent dump routine" line, saying something like "0NOTE         This data only represents the period since the previous javacore" , be an reasonable fix?

Reply:
That sounds good, but I would put it below the section separator, beginning with `1NOTE`:
```
NULL           ===============================
1NOTE          This data only represents the period since the previous javacore
```